### ISSUE TITLE
Is the theory suitable for regression problem? 

### ISSUE THREAD

**Issue by bluehalk** [putshua/ANN_SNN_QCFS#5]
Sir,
I use the clip-shift-floor to convert a network to SNN in a regression problem. I discover a phenomenon that the MSE drop to the lowest point when T=L and then rise to a regular value which is higher than ANN's loss.
( e.g. ANN validate loss = 0.03 under L = 8, SNN validate loss = 0.041 when T = 128 but SNN lowest loss = 0.028 when T = 8 = L)
I don't understand it as you do, please enlighten me.
Expecting for your reply.
